__tests__/cats.test.js

```diff
@@ -0,0 +1,87 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const Cat = require('../lib/models/Cats');
+
+describe('backend-resources routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  afterAll(() => {
+    pool.end();
+  });
+
+  it('should create a cat', async () => {
+    const res = await request(app)
+      .post('/api/v1/cats')
+      .send({ name: 'tiger', favorite_treat: 'fish', favorite_toy: 'lazer' });
```

Comment:
Use camelcase for these and in the constructer you can mutate it to snake case. Ask Dan about it in lecture, or hit me/us up for clarification if needed.